### ISSUE TITLE
error: method does not override or implement a method from a supertype

### ISSUE THREAD

**Issue by dbvt10** [Michaelvilleneuve/react-native-perspective-image-cropper#8]
Hy, the following error happened to me after rnpic install & link & build:

```
.../node_modules/react-native-perspective-image-cropper/android/src/main/java/com/reactlibrary/RNCustomCropPackage.java:19: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-perspective-image-cropper:compileReleaseJavaWithJavac FAILED
```
```
{
    "react": "16.0.0-alpha.12",
    "react-native": "0.47.1",
    "react-native-svg": "^6.0.0-rc1",
}
```
any quickfix? :)
thank you

**Comment by Michaelvilleneuve** [comment]
Simply remove `react-native-perspective-image-cropper` from your android dependencies. :)

Android is not supported yet.

**Comment by devvtms** [comment]
thnx, thats nice :)

**Comment by zedtux** [comment]
I'm adding `react-native-perspective-image-cropper` to my Android app as `react-native-document-scanner` doesn't support Android, and this project is mentioned from [the `react-native-document-scanner`'s README.md file](https://github.com/Michaelvilleneuve/react-native-document-scanner) and I'm facing this issue when compiling my Android app.

Obviously @Michaelvilleneuve your solution isn't for me 😄.

Can you please let me know how can I fix this?

**Comment by zedtux** [comment]
So based on [the Java ReactPackage](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/ReactPackage.java), removing the offending method seem to fix this issue.

The final file looks like the following :

```java

package com.reactlibrary;

import java.util.Arrays;
import java.util.Collections;
import java.util.List;

import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;
import com.facebook.react.bridge.JavaScriptModule;
public class RNCustomCropPackage implements ReactPackage {
    @Override
    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
      return Arrays.<NativeModule>asList(new RNCustomCropModule(reactContext));
    }

    @Override
    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
      return Collections.emptyList();
    }
}
```

@Michaelvilleneuve not sure if you need to manage versions or anything, otherwise I can open a PR if you'd like to. Let me know.

**Comment by Michaelvilleneuve** [comment]
@zedtux This lib does not support android either 😞

**Comment by zedtux** [comment]
Sorry @Michaelvilleneuve but I'm not following you...

By "This lib" do you mean `react-native-perspective-image-cropper` ?
If it is, then why is "this lib" suggested from the `react-native-document-scanner` documentation?

`react-native-perspective-image-cropper` is working well on my Android phone BTW.

**Comment by Michaelvilleneuve** [comment]
Nope, I wish someone could help me implement them both. In my case, I only needed iOS so I only developped the iOS version of both

**Comment by zedtux** [comment]
Okay, thank you for your replies 👍

**Comment by Michaelvilleneuve** [comment]
Fixed by 1bd80f80e919d6c00b6be9b832a1f1625495aba2 cropper is now available on both Android and iOS
